### ISSUE TITLE
tests: `pkg.sensors.TestPolicyListingWhileLoadUnload` is flaky

### ISSUE THREAD

**Issue by mtardy** [cilium/tetragon#2805]
Unfortunately we don't have artifacts since the workflow seems broken :(

https://github.com/cilium/tetragon/actions/runs/10416242015/job/28848114625?pr=2804#step:8:181

```
--- FAIL: TestPolicyListingWhileLoadUnload (10.00s)
    manager_test.go:351: adding policy
    manager_test.go:380: disabling policy
    delayed_sensor_test.go:70: unblocked failed: channel does not seem to be empty
    manager_test.go:382: 
        	Error Trace:	/home/runner/actions-runner/_work/tetragon/tetragon/go/src/github.com/cilium/tetragon/pkg/sensors/manager_test.go:382
        	Error:      	Received unexpected error:
        	            	failed to unload tracing policy "test-policy": failed to unload all sensors from collection test-policy: TestDelayedSensor/Unload timeout when waiting for unblocking
        	Test:       	TestPolicyListingWhileLoadUnload
FAIL
```

**Comment by mtardy** [comment]
oups duplicate of https://github.com/cilium/tetragon/issues/2576.
